### ISSUE TITLE
Add device detection to browser

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-core#360]
This would be useful for: 
- serving up appropriate get the app CTAs
- limiting editor features to desktop and tablet

**Comment by thisandagain** [comment]
@k88hudson Any thoughts on this?

**Comment by k88hudson** [comment]
For limiting editor features, I would recommend using screen dimensions and/or necessary feature detection as a measure of whether or not the editor will work properly;

As for the showing appropriate CTAs, it's possible (although slightly unreliable) if you think there would be a strong advantage to having different messaging for different operating systems

**Comment by xmatthewx** [comment]
I don't think this is essential for launch, but these would be better CTAs:
- Android browser - "open app"
- Desktop browser - "get the app"
- iOS browser - no CTA until we have browser editor

Totally understand that detection is messy and unreliable. Maybe there's a solid module available. Maybe not.

**Comment by k88hudson** [comment]
@xmatthewx https://github.com/faisalman/ua-parser-js is not perfect but pretty good

I agree that showing nothing on ios for the moment seems like a good idea

**Comment by ryanwarsaw** [comment]
TLDR; Need a new method in the JS interface that sends the platform type, so messages and behavior can be changed accordingly.
